src/App.js

```diff
@@ -1,68 +1,75 @@
 import React from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import { useLocation } from 'react-router';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  NavLink,
+} from 'react-router-dom';
 
 import CanvasContainer from './components/CanvasContainer';
 import './App.css';
 
-import { ExperimentList } from './experiments/';
+import experiments from './experiments/';
```

Comment:
If we REALLY wanted to get fancy, it should be possible to configure babel/webpack/etc. to actually iterate over the exports in a directory.  Similar to how we might automatically harvest all storybook stories based on a pattern.

Reply:
I figured it would take that. This is great though. Thank you.